### ISSUE TITLE
Change color on Glossary blocks

### ISSUE THREAD

**Issue by daniel-coleman-1** [ONRR/nrrd#1662]
Currently we have a light blue background with white text.
![](https://files.slack.com/files-pri/TP1E4TUF9-F029E1BCAKG/image.png)

Change text to black and background color to light gray.

**Comment by daniel-coleman-1** [comment]
https://nrrd-preview.app.cloud.gov/sites/upload-monthly-data/glossary

**Comment by daniel-coleman-1** [comment]
info.main: #d3dfe6
info.light: #e9eff2
